src/dom.js

```diff
@@ -112,6 +112,11 @@ export function documentContainsPolyfillInternal_(doc, element) {
  */
 export function removeElement(element) {
   if (element.parentElement) {
+    const activeElement = element.ownerDocument.activeElement;
+    // Restore focus to document if the focused element is inside the removed one.
```

Comment:
nit: maybe just mention accessibility as well here in case someone is wondering why it matters to move the focus to body or not.